test/MSVS/common-prefix.py

```diff
@@ -41,6 +41,12 @@
     msg = "Skipping Visual Studio test on non-Windows platform '%s'\n" % sys.platform
     test.skip_test(msg)
 
+msvs_version = '8.0'
```

Comment:
so only run this test if version 8.0 is installed?

Reply:
That was the idea.  

The change to test/MSVS/common-prefix was backed out though.  After further investigation, it doesn't matter if MSVS 8.0 is actually installed for the test to pass.

The test is specifically for generating 8.0 vcproj and sln files:
```
Test that we can generate Visual Studio 8.0 project (.vcproj) and solution (.sln) files that look correct.
```
On the other hand, it would be less work for the test suite.  It can easily be restored if desired.

Reply:
There is a pair of authors out-of-sequence alphabetically in CHANGES.TXT (i.e., Ku Kr).  

I can make the change in this PR if desired.